Elasticsearch bulk 시 SocketTime 발생

## 로그
```
2022-03-27 18:33:36,558 [I/O dispatcher 1] [DEBUG] [InternalIODispatch.timeout:114] - http-outgoing-0 [ACTIVE] Timeout
2022-03-27 18:33:36,560 [I/O dispatcher 1] [DEBUG] [RestClient.logFailedRequest:92] - request [POST http://192.168.0.18:9200/_bulk?timeout=1m] failed
java.net.SocketTimeoutException: 5,000 milliseconds timeout on connection http-outgoing-0 [ACTIVE]
	at org.apache.http.nio.protocol.HttpAsyncRequestExecutor.timeout(HttpAsyncRequestExecutor.java:387) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.client.InternalRequestExecutor.timeout(InternalRequestExecutor.java:116) [httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.InternalIODispatch.onTimeout(InternalIODispatch.java:92) [httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.client.InternalIODispatch.onTimeout(InternalIODispatch.java:39) [httpasyncclient-4.1.4.jar:4.1.4]
	at org.apache.http.impl.nio.reactor.AbstractIODispatch.timeout(AbstractIODispatch.java:175) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.BaseIOReactor.sessionTimedOut(BaseIOReactor.java:261) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.timeoutCheck(AbstractIOReactor.java:502) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.BaseIOReactor.validate(BaseIOReactor.java:211) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.AbstractIOReactor.execute(AbstractIOReactor.java:280) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.BaseIOReactor.execute(BaseIOReactor.java:104) [httpcore-nio-4.4.14.jar:4.4.14]
	at org.apache.http.impl.nio.reactor.AbstractMultiworkerIOReactor$Worker.run(AbstractMultiworkerIOReactor.java:591) [httpcore-nio-4.4.14.jar:4.4.14]
	at java.lang.Thread.run(Thread.java:750) [na:1.8.0_322]
2022-03-27 18:33:36,562 [I/O dispatcher 1] [DEBUG] [RestClient.onFailure:490] - added [[host=http://192.168.0.18:9200]] to blacklist
2022-03-27 18:33:36,562 [I/O dispatcher 1] [ERROR] [SearchService.afterBulk:327] - 5,000 milliseconds timeout on connection http-outgoing-0 [ACTIVE]
```

--- a/src/main/resources/application-default.yml
+++ b/src/main/resources/application-default.yml
@@ -36,10 +36,10 @@ jakduk:
     index-board: jakduk_dev_board
     index-gallery: jakduk_dev_gallery
     index-search-word: jakduk_dev_search_word
-    bulk-actions: 10000
+    bulk-actions: 2000
     bulk-concurrent-requests: 2
     bulk-flush-interval-seconds: 10
-    bulk-size-mb: 50
+    bulk-size-mb: 10
 
   storage:
     image-path: /jakduk/storage/image/

--- a/src/main/resources/application-dev.yml
+++ b/src/main/resources/application-dev.yml
@@ -38,10 +38,10 @@ jakduk:
     index-board: jakduk_dev_board
     index-gallery: jakduk_dev_gallery
     index-search-word: jakduk_dev_search_word
-    bulk-actions: 10000
+    bulk-actions: 2000
     bulk-concurrent-requests: 2
     bulk-flush-interval-seconds: 10
-    bulk-size-mb: 50
+    bulk-size-mb: 10
 
   storage:
     image-path: /jakduk/storage/image/

--- a/src/main/resources/application-prod.yml
+++ b/src/main/resources/application-prod.yml
@@ -36,10 +36,10 @@ jakduk:
     index-board: jakduk_prd_board
     index-gallery: jakduk_prd_gallery
     index-search-word: jakduk_prd_search_word
-    bulk-actions: 10000
+    bulk-actions: 2000
     bulk-concurrent-requests: 2
     bulk-flush-interval-seconds: 10
-    bulk-size-mb: 50
+    bulk-size-mb: 10
 
   storage:
     image-path: /jakduk/storage/image/

